build/enso-formatter/src/main.rs

```diff
@@ -284,8 +286,10 @@ fn print_h2(
     tokens: &[HeaderToken],
     str: &str,
 ) {
+    use std::fmt::Write;
+
     if tokens.iter().map(|tok| map.contains_key(tok)).any(|t| t) {
-        out.push_str(&format!("// === {} ===\n", str));
+        writeln!(out, "// === {} ===", str).unwrap()
```

Comment:
I don't like these changes TBH. They are using `unwrap` now and no unwrap in the previous version. Why these changes?